Lib/logging/__init__.py

```diff
@@ -1244,6 +1244,17 @@ def _fixupChildren(self, ph, alogger):
                 alogger.parent = c.parent
                 c.parent = alogger
 
+    def _clear_cache(self):
+        """
+        Clear the cache for all loggers in loggerDict
+        Called when level changes are made
+        """
+
+        _acquireLock()
```

Comment:
Oh ok, right, I first understood that the lock was supposed to protect the cache :-) I don't think that the cache deserves a lock.